tests/active_features/env-deploy.feature

```diff
@@ -22,13 +22,16 @@ Feature: Site Deployment
     When I run "terminus env:deploy test --site=[[test_site_name]] --sync-content --note='Deploy test'"
     Then I should get: "There is nothing to deploy."
 
-  @vcr site_init-env
-  Scenario: Initializing test when it has not been previously initialized
-    When I run "terminus env:deploy test --site=[[test_site_name]] --sync-content --note='First deploy to live'"
-    Then I should get "."
-    And I should get "."
-    Then I should get:
-    """
-    Deploying code to "test", and cloning files from "live", and cloning database from "live"
-    """
-    
+  # This will only work with a fixture for API call:
+  #   https://onebox/api/sites/1fdf3bf6-50e3-42d8-ae56-fb4051481404/environments/dev/code-log
+  # Currently the fixture only contains `.../test/code-log`
```

Comment:
@marktheunissen I'm confused. The fixture `site_deploy_no_changes` has no API calls other than the login ones and .../code-log, but the fixture that has the call to `https://onebox/api/sites/1fdf3bf6-50e3-42d8-ae56-fb4051481404/environments/dev/code-log` in it is `site_init-env` which also contains several other calls..